bluechi.spec.in

```diff
@@ -1,3 +1,7 @@
+# python bluechi module is enabled by default, it can be disabled passing `--without python` option to rpmbuild
```

Comment:
Please make this a boolean (like we have in the hirte spec file), it will be a lot easier for package maintenance as it will allow to have it enabled in Fedora and disabled in AutoSD and leverage git to merge changes from Fedora into AutoSD without having to worry about enabling it suddenly.

Reply:
OK, change in https://github.com/containers/bluechi/pull/480

I didn't know you are already using similar approach in Fedora, it would be really great to convert multiple different spec files for various platforms into this upstream one and then only pass on each build platform different parameters as needed

@pypingou